eZ/Bundle/EzPublishCoreBundle/DependencyInjection/Configuration/Parser/Common.php

```diff
@@ -168,6 +168,7 @@ public function mapConfig(array &$scopeSettings, $currentScope, ContextualizerIn
             $contextualizer->setContextualParameter('binary_dir', $currentScope, $scopeSettings['binary_dir']);
         }
         if (isset($scopeSettings['api_keys']['google_maps'])) {
+            $contextualizer->setContextualParameter('api_keys', $currentScope, ['google_maps' => $scopeSettings['api_keys']['google_maps']]);
             $contextualizer->setContextualParameter('api_keys.google_maps', $currentScope, $scopeSettings['api_keys']['google_maps']);
```

Comment:
This is somewhat confusing to me, because this line, without the one added above, works for me. I set api key as [described in the Configuration section of our doc](https://doc.ez.no/display/DEVELOPER/MapLocation+Field+Type) and I can see it properly appended in the rendered view (on front-end). About test cases, see next comment below.